### DISCUSSION TITLE
Endpoint information in getSecurityServerOperationalData

### DISCUSSION THREAD

**Discussion by fredycuello** [nordic-institute/X-Road-Metrics#107] [Ideas]
Service getSecurityServerOperationalData retrieves information only at the Service level, but if the service has multiple endpoints, there is no possibility of measuring the utilization of each endpoint directly. It is only possible by recovering each of the messages and parsing the content of the request.
It would be nice if the log record contained the field indicating the endpoint used.

**Comment by petkivim** [comment]
Hi @fredycuello! Thanks for sharing the idea! As a background information, the `getSecurityServerOperationalData` service was originally designed and implemented when X-Road only supported SOAP. It wasn't updated when the REST support was implemented and therefore, it doesn't know anything about REST endpoints. 

Your enhancement request will be discussed at the next X-Road Technical Committee meeting in January 9. I'll let you know the outcome after the meeting.

**Comment by petkivim** [comment]
Hi @fredycuello! I'm glad to inform you that the Technical Committee approved this enhancement request. Next, we'll create an X-Road backlog item for it and it'll be prioritised together with other backlog items.

**Comment by petkivim** [comment]
This feature has been included in [X-Road version 7.6.0](https://nordic-institute.atlassian.net/wiki/spaces/XRDKB/pages/833716225/X-Road+v7.6.0+Release+Notes). More information about the implementation is available [here](https://nordic-institute.atlassian.net/browse/XRDDEV-2567).
